CONTRIBUTING.md

```diff
@@ -55,15 +55,16 @@ To make the contribution process as seamless as possible, we ask for the followi
 #### Environment
 
 - Fork the repository.
-- Clone your forked repository locally.
-- We use Go Modules to manage dependencies, so you can develop outside your `$GOPATH`.
-- We use [golangci-lint](https://github.com/golangci/golangci-lint) to lint our code, you can install it locally via `make setup`.
+- Clone your fork locally.
+- Set up development environment:
+  - We use Go Modules to manage dependencies, so you can develop outside your `$GOPATH`.
+  - Run `make setup` to install required dependencies and developer tools. This includes [golangci-lint](https://github.com/golangci/golangci-lint), which we use to lint our code.
```

Comment:
should we also mention any of those two commands:
```
deps                           Download go module dependencies
devtools                       Install dev tools
```

Reply:
Looking into the makefile, `make setup` already runs deps, devtools and link-git-hooks so I don't think it is needed.

Reply:
ah, perfect! thanks